### PR TITLE
8336035: Change JavaFX release version to 23.0.1 in jfx23u

### PR DESCRIPTION
**Pull request by kevinrushforth**
Updates for the beginning of the 23.0.1 release.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] [JDK-8336035](https://bugs.openjdk.org/browse/JDK-8336035) needs maintainer approval

### Issue
 * [JDK-8336035](https://bugs.openjdk.org/browse/JDK-8336035): Change JavaFX release version to 23.0.1 in jfx23u (**Task** - P2 - Approved)

### Reviewers
 * [Ambarish Rapte](https://openjdk.org/census#arapte) (@arapte - **Reviewer**)
 * [Johan Vos](https://openjdk.org/census#jvos) (@johanvos - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx23u.git pull/1/head:pull/1` \
`$ git checkout pull/1`

Update a local copy of the PR: \
`$ git checkout pull/1` \
`$ git pull https://git.openjdk.org/jfx23u.git pull/1/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1`

View PR using the GUI difftool: \
`$ git pr show -t 1`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx23u/pull/1.diff">https://git.openjdk.org/jfx23u/pull/1.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jfx23u/pull/1#issuecomment-2224089422)

### CODE REVIEW

**Comment by kevinrushforth** [comment]
Reviewers: @johanvos or @arapte

**Comment by kevinrushforth** [comment]
/approval request This is the required "start of release" version bump. NOTE: This fix must be integrated before any other fix (meaning before the repo is announced as being open for accepting backports).

**Comment by kevinrushforth** [comment]
@johanvos In addition to reviewing this, can you add the needed maintainer approval, either using the Skara /approve command or directly in JBS? No hurry, since I do not plan to announce that this repo is open until some time next week.

**Review by arapte — APPROVED** [review]

**Review by johanvos — APPROVED** [review]

**Pull request closed without merging**
